PRNTRS Vendor/Vendor Sign-In/views/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+
+    <meta charset="utf-8">
+    <title>Vendor Sign In</title>
+    <link rel="stylesheet" href="styles.css">
```

Comment:
The `href` links to `styles.css` but the `styles.css` file is moved to the `public` directory from what I can see here, so shouldn't the path be different?
What am I missing here @vedanthv ?

Reply:
@A-HK You can read why I have done that in the code review here: https://github.com/Tech-Matrix/PRNTRS/pull/28